main.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/humio/humio-operator/pkg/helpers"
 	"github.com/humio/humio-operator/pkg/humio"
 
+	corev1alpha1 "github.com/humio/humio-operator/api/v1alpha1"
```

Comment:
```suggestion
```

Reply:
We end up registering the same thing twice, that is probably not the intent here.